charts/crds/quoteattestations.tcs.intel.com.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: quoteattestations.tcs.intel.com
```

Comment:
Are these files duplicated? I think this file or something very close to it is already in `config/crd/tcs.intel.com_quoteattestations.yaml`. Maybe the Helm charts could reuse those or the other way around? I'm thinking if it would be possible to say `make helm` or similar which could generate these files?

Reply:
`make helm` implemented in the updated pach